src/GogeDao.sol

```diff
@@ -370,718 +476,955 @@ contract GogeDAO is Ownable {
-        require(newOwner != address(0));
-        super.transferOwnership(newOwner);
+    /// @notice An internal method for updating minPollPeriod value.
+    /// @param  _amount minPollPeriod value.
+    function _updateMinPollPeriod(uint256 _amount) internal {
+        minPeriod = _amount;
     }
 
-    function payTeam() public {
-        uint256 amount = teamBalance.div(teamMembers.length);
-        for(uint256 i = 0; i < teamMembers.length.sub(1); i++) {
-            payable(teamMembers[i]).transfer(amount);
-            teamBalance -= amount;
-        }
-        payable(teamMembers[teamMembers.length.sub(1)]).transfer(teamBalance);
+    /// @notice An internal method for updating governanceTokenAddr.
+    /// @param  _addr new governance token address.
+    function _changeGovernanceToken(address _addr) internal {
+        governanceTokenAddr = _addr;
     }
 
-    // ---------- Mutative -----------
+    // ----
+    // View
+    // ----
 
-    function _updateVetoEnabled(bool enabled) internal {
-        require(veto != enabled, "Already set");
-        veto = enabled;
-        emit VetoEnabledUpdated(enabled);
+    /// @notice A view method for returning the amount of votes of a voter in a poll.
+    /// @param  _addr address of voter.
+    /// @param _pollNum unique poll identifier.
+    function getVotes(address _addr, uint256 _pollNum) public view returns (uint256) {
+        return polls[_pollNum][_addr];
     }
 
-    function updateVetoEnabled(bool enabled) external onlyOwner() {
-        _updateVetoEnabled(enabled);
+    /// @notice A view method for returning a poll's unique metadata.
+    /// @param _pollNum unique poll identifier.
+    function getMetadata(uint256 _pollNum) public view returns (Metadata memory) {
+        return pollMap[_pollNum];
     }
 
-    function setTeamMember(address addr, bool value) internal {
-        if(value) {
-            (bool _isTeamMember, ) = isTeamMember(addr);
-            if(!_isTeamMember) teamMembers.push(addr);        
-        } else {
-            (bool _isTeamMember, uint8 s) = isTeamMember(addr);
-            if(_isTeamMember){
-                teamMembers[s] = teamMembers[teamMembers.length - 1];
-                teamMembers.pop();
-            } 
+    /// @notice A view method for returning a poll's current proportion of votes over circuating supply.
+    /// @param _pollNum unique poll identifier.
+    function getProportion(uint256 _pollNum) public view returns (uint256) {
+        return totalVotes[_pollNum] * 100 / IGogeERC20(governanceTokenAddr).getCirculatingMinusReserve();
+    }
+
+    /// @notice A view method for returning the current polls (by pollNum) are active, but expired.
+    /// @return expired array of pollNums that are active, but expired.
+    /// @return counter the amount of polls that are active, but expired.
+    function findExpiredPolls() public view returns (uint256[] memory expired, uint256 counter) {
```

Comment:
I've voiced my concerns about this method in the `queryEndTime()` function. Why is this function necessary on its own?

Reply:
I guess it's not at least in the contract. I do think this could be useful as an external call say an admin wants to query this and see which polls exist that are expired that need to be removed

Reply:
I can elaborate more on this when we meet